docs/component/identity-wallet.md

```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
 sidebar_label: Identity Mobile App
 ---
 
-Overview
+OA provides it's own mobile application to view, store and share Verifiable Document and Transferable Records.
 
-## Header
+## Installation
+
+For the moment, the application is not available on Apple store or Android store. You will need to run it through [Expo](https://expo.io/).
+
+1. Download Expo [![Expo on apple store](/docs/component/identity-wallet/apple-store.svg)](https://apps.apple.com/fr/app/expo-client/id982107779 "Open expo on apple store") [![Expo on apple store](/docs/component/identity-wallet/android-store.svg "Open expo on android store")](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en)
```

Comment:
maybe use US link instead? [https://apps.apple.com/us/app/expo-client/id982107779]

Reply:
why US ? this one is without any region :thinking:

Reply:
oh, i thought that the app page itself in english might be understandable, currently is in french.

<img width="768" alt="Screenshot 2020-02-21 at 9 35 13 AM" src="https://user-images.githubusercontent.com/4774314/74996091-9844f780-548d-11ea-931d-20cfa3e65301.png">

Reply:
how come it's in french .....................

Reply:
oh yeah there is the fr I didnt notice thanks

Reply:
Changed to https://apps.apple.com/app/expo-client/id982107779 => no lang